src/AbstractConstCollectionArray.php

```diff
@@ -120,38 +125,66 @@ public function jsonSerialize()
         return $this->container;
     }
 
+    /**
+     * @return ArrayList
+     */
     public function toVector()
     {
         return new ArrayList($this);
     }
 
+    /**
+     * @return ImmArrayList
+     */
     public function toImmVector()
     {
         return new ImmArrayList($this);
     }
 
+    /**
+     * TODO: Implement toSet() method.
+     */
     public function toSet()
     {
-        // TODO: Implement toSet() method.
     }
 
+    /**
+     * @return ImmSet
+     */
     public function toImmSet()
     {
         return new ImmSet($this);
     }
 
+    /**
+     * @return LazyIterableView
+     */
     public function lazy()
     {
         return new LazyIterableView($this);
     }
 
+    /**
+     * @return Dictionary
+     */
     public function toMap()
     {
         return new Dictionary($this);
     }
 
+    /**
+     * @return ImmDictionary
+     */
     public function toImmMap()
     {
         return new ImmDictionary($this);
     }
+
+    /**
+     * @return array
+     */
+    public function getAll()
```

Comment:
This would not be necessary. The container should not be exposed through any kind of method. Once this is an *Iterable* it's quite ok to not expose it